pipelinewise/fastsync/commons/tap_postgres.py

```diff
@@ -332,7 +332,7 @@ def fetch_current_incremental_key_pos(self, table, replication_key):
         result = self.query(
             f'SELECT MAX({replication_key}) AS key_value FROM {schema_name}."{table_name}"'
         )
-        if len(result) == 0:
+        if not result:
```

Comment:
I could not find any unit test for this method. It could be nice to implement a unit test to cover at least these new implementations

Reply:
unit tests added